lib/ts/normalisedURLPath.ts

```diff
@@ -59,7 +64,7 @@ function normaliseURLPathOrThrowError(input: string): string {
         !input.startsWith("https://")
     ) {
```

Comment:
we should not change this class at all. It's being used in lots of places. Instead, we should separately handle query params and make sure that that works for the redirectToPath